src/app/parcels/ParcelsPage.tsx

```diff
@@ -56,6 +57,32 @@ const defaultShownHeaders: (keyof ParcelsTableRow)[] = [
     "lastStatus",
 ];
 
+const sortStatusByWorkflowOrder = (
+    statusA: ParcelsTableRow["lastStatus"],
+    statusB: ParcelsTableRow["lastStatus"]
+): number => {
+    const indexA = statusNamesInWorkflowOrder.indexOf(statusA?.name ?? "-");
+    const indexB = statusNamesInWorkflowOrder.indexOf(statusB?.name ?? "-");
```

Comment:
I think the default value can be any string that's not in the array - `indexOf` would return -1 if the string is not found.

Reply:
That makes sense thanks